src/runtime.js

```diff
@@ -135,9 +135,9 @@ export function adopt(global) {
    * @param {function()=} opt_flush
    * @export
    */
-  global.AMP.setTickFunction = (fn, opt_flush) => {
+  global.AMP.setTickFunction = (fn, opt_flush, opt_setParams) => {
```

Comment:
@ericfs can you enumerate/confirm the actual event names you want for these?

  - 'tick'
  - 'flush'
  - 'setParams'